redaxo/src/core/pages/setup.php

```diff
@@ -681,9 +681,11 @@
 
     $n = [];
     $n['label'] = '<label for="rex-form-redaxo-user-pass">' . rex_i18n::msg('setup_608') . '</label>';
-    $n['field'] = '<input class="form-control rex-js-redaxo-user-pass" type="password" value="' . $redaxo_user_pass . '" id="rex-form-redaxo-user-pass" name="redaxo_user_pass" />';
+    $n['field'] = '<input class="form-control rex-js-redaxo-user-pass" type="password" value="' . $redaxo_user_pass . '" id="rex-form-redaxo-user-pass" name="redaxo_user_pass" />
```

Comment:
@gharlan  hmm hier fehlt escaping wg. XSS (und passwörter mit `<>` scheinen aktuell nicht möglich??
(gleiches für den login paar zeilen drüber)

Reply:
Ja, hier scheint das Escaping in der Tat zu fehlen.

Reply:
https://github.com/redaxo/redaxo/pull/1413